ksqldb-engine/src/test/java/io/confluent/ksql/connect/supported/JdbcSourceTest.java

```diff
@@ -21,7 +21,7 @@
 import com.google.common.collect.ImmutableMap;
 import io.confluent.ksql.connect.Connector;
 import io.confluent.ksql.metastore.model.DataSource.DataSourceType;
-import io.confluent.ksql.metastore.model.MetaStoreMatchers.OptionalMatchers;
+import io.confluent.ksql.test.util.OptionalMatchers;
```

Comment:
I moved this matcher up the project dependency graph so I could use it in another test.